packages/runtime/container-runtime/src/channelCollection.ts

```diff
@@ -512,7 +517,7 @@ export class ChannelCollection implements IFluidDataStoreChannel, IDisposable {
 			return false;
 		}
 
-		this.parentContext.addedGCOutboundRoute("/", `/${internalId}`);
+		this.parentContext.addedGCOutboundRoute("/", `/${internalId}`, messageTimestampMs);
```

Comment:
Per other comment, I think we should just not call this if `messageTimestampMs` is undefined.

Reply:
Hmmmmm... trying to think it through some more.  As-is in the PR, GC won't write down this added route if timestamp is missing.  That's a change from before.

In the detached case, the alias will be found when GC first runs after it's attached.  Ahh, and GC will be told about the route when processing the attach op.  So I think we're ok.  And I do think it's more clear to skip calling this here rather than bailing early way down in GC (and having to support missing timestamp on all these functions)

Reply:
Yes, that sounds right. The detached case is when container is detached. GC doesn't even run in that container so not notifying it should be fine. I will make that change.